src/core/style/expressions/palettes.js

```diff
@@ -1,5 +1,31 @@
+
 import * as cartocolor from 'cartocolor';
 
+/**
+ * ### Color palettes
+ * 
+ * Palettes are constants that allow to use {@link https://carto.com/carto-colors/|cartocolors} easily.
+ * Use them with a {@link carto.style.expressions.ramp|ramp}
+ * 
+ * The following palettes are availiable:
+ *  - Categorical:
+ *      - PRISM
+ *      - EARTH
+ *  - Numeric
+ *      - ...
```

Comment:
Where are the numeric palettes?

Reply:
[here](https://github.com/CartoDB/CartoColor/blob/master/cartocolor.js) tagged as cuantitative, names are going to change so I prefer to wait for a decision before copying the full list

Reply:
Please, change this, we may add true numeric palettes in the future (list of numbers for bubble maps), and there are multiple "numeric" palette types I think